internal/service/hadoop/hadoop.go

```diff
@@ -109,13 +109,13 @@ func (r *hadoopResource) Schema(_ context.Context, req resource.SchemaRequest, r
 						stringvalidator.LengthBetween(8, 20),
 						stringvalidator.RegexMatches(regexp.MustCompile(`[A-Z]+`), "Must have at least one uppercase alphabet"),
 						stringvalidator.RegexMatches(regexp.MustCompile(`\d+`), "Must have at least one number"),
-						stringvalidator.RegexMatches(regexp.MustCompile(`[~!@#$%^*()\-_=\[\]\{\};:,.<>?]+`), "Must have at least one special character"),
-						stringvalidator.RegexMatches(regexp.MustCompile(`^[^&+\\"'/\s`+"`"+`]*$`), "Must not have ` & + \\ \" ' / and white space."),
+						stringvalidator.RegexMatches(regexp.MustCompile(`[~!@#$%^*+()\-_=\[\]\{\};:,.<>?]+`), "Must have at least one special character"),
```

Comment:
how about replace it like `[^a-zA-Z0-9]+` instead of list of all special characters? I found that the "|" character is missing from the list also.

Reply:
fix

Reply:
`\W` meta character exclude underscore(_), does it intended change?

Reply:
No, I was referring to the wrong place in the regular expression description. Fixed.